stdlib/2and3/xml/dom/__init__.pyi

```diff
@@ -0,0 +1,67 @@
+class Node:
+    ELEMENT_NODE: int
```

Comment:
For class-level variables, use `= ...`. Leave it off for module-level constants and class instance variables. For example, look at https://github.com/python/typeshed/blob/c50fce6ef3111bba4d0b46e3c1f8e846d05cf0c3/stdlib/2and3/bdb.pyi#L69

Reply:
It's actually optional whether to have `= ...` for module-level constants, but from what I remember the encouraged style is to leave them off.